<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1599</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Ekran" name="ekran" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>10</y>
      <width>271</width>
      <height>721</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="trX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="trY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="trZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotY">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>270</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotZ">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>310</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="shX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="shY">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>430</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="shZ">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>470</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Tranformation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Shearnig</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>530</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QSlider" name="scX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>560</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scY">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>600</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scZ">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>640</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="nrSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1599</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Ekran</class>
   <extends>QWidget</extends>
   <header>ekran.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
